ddtrace/context.py

```diff
@@ -4,6 +4,7 @@
 from typing import Optional
 from typing import TYPE_CHECKING
 from typing import Text
+from typing import Tuple
```

Comment:
nit: we can move this line under the `if TYPE_CHECKING:` check

Reply:
oh sure